atomic_reactor/auth.py

```diff
@@ -146,12 +173,15 @@ def __call__(self, request):
         url_parts = urlparse(request.url)
 
         if self.V1_URL.search(url_parts.path):
-            if not self.username or not self.password:
+            if (not self.username or not self.password) and not self.auth_b64:
                 # V1 API only supports basic auth which requires user/pass
```

Comment:
Is this comment still valid? IIUC we cannot use `auth` with v1

Reply:
Ahh nevermind, it is still basic auth

Reply:
I believe the comment is valid, although V1 only API support has been dropped in osbs-client. I believe that this whole code snippet will have to go at some point.